### ISSUE TITLE
Recovering from a crash.

### ISSUE THREAD

**Issue by AdamMagaluk** [zettajs/zetta#168]
Running the below snippet produces the issue. Every other time ran it will not show any devices provisioned or peers connections. Providing a callback to .listen() you can see an error below.

```js
zetta()
  .use(LED)
  .use(function() {
    throw new Error('123')
  })
  .listen(1337, function(err) {

  });
```

`err` produced
```
{ [Error: ENOENT, unlink '/Users/ApigeeCorporation/Software/zetta/zetta/sample/multiple-peers/detroit/.devices/214.medea.hint']
  errno: 34,
  code: 'ENOENT',
  path: '/Users/ApigeeCorporation/Software/zetta/zetta/sample/multiple-peers/detroit/.devices/214.medea.hint' }
```

This particular error is a result of medea compaction throwing an error when a missing hint file is trying to be deleted.

Compact tries to remove the hint file: https://github.com/medea/medea/blob/63169f2995e969d7d4f606b6543927bd9f99fcb6/compactor.js#L164

It's caught in zetta here and bubbles up to zetta.listen. https://github.com/zettajs/zetta/blob/daab94e6aaac0580604e1094331fba245ff3d802/lib/registry.js#L63-L64

**Comment by kevinswiber** [comment]
@kyork-cl I think you were experiencing this yesterday.

**Comment by kevinswiber** [comment]
@AdamMagaluk Can you file an issue on [medea/medea](/medea/medea), and we'll track it here?

**Comment by AdamMagaluk** [comment]
@kevinswiber Issue in medea https://github.com/medea/medea/issues/54

**Comment by kevinswiber** [comment]
Fixed by medea@0.17.0, medeadown@1.1.6.  Thanks @AdamMagaluk!  As soon as we get a PR bumping up to those versions, we can merge and release.
